libs/api/metadata-converter/src/lib/iso19139/read-parts.spec.ts

```diff
@@ -744,6 +745,30 @@ describe('read parts', () => {
           ])
         })
       })
+
+      describe('read resource identifier', () => {
```

Comment:
I think it can be removed, and actually I'm not sure it's actually working as intended (as the provided XML does not include the full hierarchy i.e. identification block, citation block...)